charts/argocd-apps/templates/events-application.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: events
+  namespace: argocd
+spec:
+  destination:
+    namespace: events
+    server: {{ .Values.spec.destination.server }}
+  project: default
+  source:
+    helm:
+      valueFiles:
+      - values.yaml
```

Comment:
Could we use a `{{- if .Values.dev.enabled -}}` to use the `dev-values` for each app?